assets/blocks/core-image/loader.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Newspack Core Image, Loader
+ */
+
+/**
+ * Dependencies
+ */
+// WordPress
+import { Spinner } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+/**
+ * Types
+ */
+import { AttributesMeta, AttributeProps } from './types';
+
+const Loader = ( { attributes, setAttributes }: AttributeProps ) => {
+	const imageId = attributes.id ?? 0;
+	const { lockPostSaving, unlockPostSaving } = useDispatch( 'core/editor' );
+	const { meta }: { meta: AttributesMeta } = useSelect(
+		select => select( 'core' ).getMedia( imageId ),
+		[ imageId ]
+	) ?? {
+		meta: {},
+	};
+	useEffect( () => {
+		if ( Object.keys( meta ).length ) {
+			setAttributes( { meta } );
```

Comment:
Sorry, this slipped my previous review: not _all_ meta values should be saved. We're only interested in the credit-related metadata. Otherwise, the block attributes will be polluted with unneeded meta:

<img width="755" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/7383192/d9faf21b-70db-40cb-b047-dda1684f281b">

Reply:
@adekbadek This was by design in an effort to eliminate future needs for future attachment meta in distributed posts. 

I have filtered out all meta except for `_media_credit`, `_media_credit_url`, `_navis_media_credit_org` (Required to recreate the credit on distributed posts), in a4f0f53274e74cc46f341dd90f70cc7c342c0bdb